src/Components/Search/Search.css

```diff
@@ -3,11 +3,12 @@
   justify-content: flex-end;
   align-items: flex-end;
   margin-right: 1rem;
+  height: 100px;
 }
 
 .search__input {
```

Comment:
The Navbar covers the 'Selector' logo header a little bit once it reaches mobil view size, make a little smaller or add margin possibly